src/app/pages/datasets/components/dataset-details-panel/dataset-details-panel.component.ts

```diff
@@ -34,6 +34,45 @@ export class DatasetDetailsPanelComponent implements OnInit {
     });
   }
 
+  get datasetHasRoles(): boolean {
```

Comment:
Just wondering if can be used in other places and if it is worth moving it to `src/app/pages/datasets/utils/dataset.utils.ts` like other helper functions.

Reply:
We just introduced this terminology and this only applies in the context of the datasets page. `Roles` doesn't really have a technical meaning. Can't think of a reason for it to be reused.

Reply:
I was thinking of `Roles` columns in the tree if these conditions can be helpful there.